From 911cbb961bf97013cdf447feb310635da36641ba Mon Sep 17 00:00:00 2001
From: Wentao Guan <guanwentao@uniontech.com>
Date: Fri, 21 Feb 2025 18:41:49 +0800
Subject: [PATCH 323/360] deepin: wifi: mt76: mt7921: add pci aspm probe log

We met many platform relate issue,
add the log to help tracking aspm bug.

Signed-off-by: Wentao Guan <guanwentao@uniontech.com>
---
 drivers/net/wireless/mediatek/mt76/mt7921/pci.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/net/wireless/mediatek/mt76/mt7921/pci.c b/drivers/net/wireless/mediatek/mt76/mt7921/pci.c
index 67723c22aea6..3765bf6609a4 100644
--- a/drivers/net/wireless/mediatek/mt76/mt7921/pci.c
+++ b/drivers/net/wireless/mediatek/mt76/mt7921/pci.c
@@ -343,6 +343,9 @@ static int mt7921_pci_probe(struct pci_dev *pdev,
 	if (!mt7921_disable_aspm && mt76_pci_aspm_supported(pdev))
 		dev->aspm_supported = true;
 
+	pr_info("mt7921_pci_probe: disable_aspm:%d,aspm_supported:%d\n",
+		mt7921_disable_aspm, dev->aspm_supported);
+
 	ret = mt792xe_mcu_fw_pmctrl(dev);
 	if (ret)
 		goto err_free_dev;
-- 
2.20.1

